src/manual-edits/content/docs/new-relic-only/gatsby-configuration/create-dummy-docs.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: 'Making dummy docs'
+tags:
+translate:
+redirects:
+---
+
+If you have a document that applies to multiple categories, you can create something that is akin to the dummy docs that we used to make in Drupal.
```

Comment:
We should be aware of our audience, which is now everyone not just writers, and not make references to drupal like this.